templates/types/streaming/fastapi/app/api/services/file.py

```diff
@@ -78,25 +81,25 @@ def store_and_parse_file(file_data, extension) -> List[Document]:
         return documents
 
     @staticmethod
-    def process_file(base64_content: str) -> List[str]:
+    def process_file(file_name: str, base64_content: str) -> List[str]:
         file_data, extension = PrivateFileService.preprocess_base64_file(base64_content)
-        documents = PrivateFileService.store_and_parse_file(file_data, extension)
-
-        # Only process nodes, no store the index
-        pipeline = IngestionPipeline()
-        nodes = pipeline.run(documents=documents)
 
         # Add the nodes to the index and persist it
+        # TODO: get params to select index
         current_index = get_index()
```

Comment:
@thucpn here we're not passing the index selected in the UI

Reply:
@thucpn great catch - the problem is actually that we import LlamaCloudFileService every time - I think the code should not even exist if we don't select LlamaCloud - I'll have a look